app/controllers/mou_controller.rb

```diff
@@ -7,8 +7,14 @@ def index
   def create
     redirect_path = replacing? ? replaced_mou_index_path : created_mou_index_path
     if params[:signed_mou]
-      current_organisation.signed_mou.attach(params[:signed_mou])
-      flash[:notice] = "MOU uploaded successfully."
+      mime_type = Marcel::MimeType.for(params[:signed_mou])
```

Comment:
the mime type is already available in `params[:signed_mou].content_type`.

Reply:
done